feeds/combiner.go

```diff
@@ -289,24 +311,24 @@ func (cf *combinerFeed) loop() {
 	}
 }
 
-//
-// Helper Methods
-//
-
 // subscriberInfoToHeaders converts subscriber information to map[string]string
-func subscriberInfoToHeaders(subscriber *Subscriber) map[string]string {
+func subscriberInfoToHeaders(subscriber *domain.Subscriber) map[string]string {
 	return map[string]string{
 		"bot-id":    subscriber.BotID,
 		"bot-owner": subscriber.BotOwner,
 	}
 }
 
 // isSameSubscription checks for field-for-field equality for subscriptions.
-func isSameSubscription(a, b *CombinerBotSubscription) bool {
+func isSameSubscription(a, b *domain.CombinerBotSubscription) bool {
```

Comment:
added subscriber info to compare method, because now that we have fees, no subscription is equal anymore